google-cloud-errors/lib/google/cloud/errors/version.rb

```diff
@@ -16,7 +16,7 @@
 module Google
   module Cloud
     module Errors
-      VERSION = "1.1.0".freeze
+      VERSION = "1.2.0".freeze
```

Comment:
Revert this change. Release-please will update the version.

Reply:
:+1: